src/dbnode/storage/bootstrap/bootstrapper/peers/source.go

```diff
@@ -958,7 +958,13 @@ func (s *peersSource) processReaders(
 			blockStart,
 			blockEnd,
 		)
-		if err != nil {
+		if errors.Is(err, fs.ErrOutOfRetentionClaim) {
```

Comment:
Any tests for `peers/source.go` changes?

Reply:
Added out of retention test for peers